### DISCUSSION TITLE
We need more information

### DISCUSSION THREAD

**Discussion by dwgong** [community/community#52548] [GitHub Education] [answered]
<img width="1049" alt="Screenshot 2023-04-11 at 14 21 53" src="https://user-images.githubusercontent.com/127175558/231254706-9721e4f2-1603-4ba9-8b90-9e59a5db027f.png">
I tried to verify my student status with this unofficial transcript (The black parts are my ID and grades that are not covered in a real submission). My account name has been set to my full name. But I still get this email saying the academic affiliation couldn't be verified.
<img width="600" alt="Screenshot 2023-04-11 at 14 23 37" src="https://user-images.githubusercontent.com/127175558/231255024-732e1de0-ce60-4221-990d-cfddff8fe1ad.png">

**Comment by samus-aran** [comment, marked as answer]
Hi @dwgong 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
